drivers/media/i2c/imx477.c

```diff
@@ -112,6 +112,9 @@ MODULE_PARM_DESC(trigger_mode, "Set vsync trigger mode: 1=source, 2=sink");
 #define IMX477_REG_XVS_IO_CTRL		0x3040
 #define IMX477_REG_EXTOUT_EN		0x4b81
 
+/* Temperature sensor */
+#define IMX477_REG_TEMP_SEN_CTL		0x0138
```

Comment:
As you've defined this, it'd be nice to use that define in `mode_common_regs` so all instances of 0x0138 are by name.

Reply:
There are currently no names used in mode_common_regs, and counterexamples where suitable names exist (IMX477_REG_LINE_LENGTH, IMX477_REG_LINE_LENGTH, ...).